en_US.json

```diff
@@ -1064,17 +1071,26 @@
 	"hint.directive.proxy_pass_request_headers": "pass request headers from client to upstream",
 	"hint.directive.proxy_redirect": "replace the 30x Location header",
 	"hint.directive.proxy_request_body_in_cache_key": "add req body in cacke key",
+	"hint.directive.proxy_request_buffering": "enable or disable request buffering",
 	"hint.directive.proxy_set": "assign value to a variable based on response",
 	"hint.directive.proxy_ssl_protocols": "TLS protocols to origin",
+	"hint.directive.range_reorder": "reorder and coalesce multiple ranges",
 	"hint.directive.realtime_log_downsample": "set the downsample factor",
 	"hint.directive.sanitize_accept_encoding": "preprocess Accept-Encoding header",
 	"hint.directive.satisfy": "how to combine multiple auth results",
 	"hint.directive.secure_link": "set up a built-in access control method",
 	"hint.directive.secure_link_md5": "set up a built-in access control method",
 	"hint.directive.secure_link_secret": "set up a built-in access control method",
+	"hint.directive.set_attack_state_cpu": "sets threshold for specified $server_name and random number between mintime and maxtime as expiration for resetting of the variable attack_state_cpu state",
+	"hint.directive.set_attack_state_req": "sets threshold for specified server and random number between a range mintime and maxtime as expiration for resetting of the variable attack_state_req state",
+	"hint.directive.set_attack_state_rspbytes": "sets threshold for specified server and random number between a range mintime and maxtime as expiration for resetting of the variable",
+	"hint.directive.set_attack_state_tlshs": "sets threshold for specified server and random number between a range mintime and maxtime as expiration for resetting of the variable attack_state_tlshs state",
```

Comment:
Can you please share where did you get the descriptions from?
Is it mentioned somewhere?

Reply:
I referred to the Cache Operation Handbook (https://docs.google.com/document/d/1T4NVOiiv_OlYA6nzDcoTm7MpQMBz5E1nr-W78_7GNiQ/edit#). Should I officially receive the description separately? If I have to, I will request the reporter through the issue comment.

Reply:
I will request a description. So I'm going to close this PR.

Reply:
I asked Reporter Jianghai Yao for a description.
Thank you for pointing that out. @chiraags